artifactory.py

```diff
@@ -245,6 +245,13 @@ def chunks(data, size):
         yield {k: data[k] for k in islice(it, size)}
 
 
+def print_download_progress(offset, total_size, *args, **kwargs):
+    msg = "Downloaded {}/{}MB...[{}%]".format(int(offset / 1024 / 1024),
+                                              int(total_size / 1024 / 1024),
+                                              round(offset / total_size * 100, 2))
+    print(msg)
```

Comment:
`print` doesn't seem like a part of library. We should use `logging.debug` if we want to show the progress

Reply:
Hi,
Thanks for reply

Sure, we can do it with logging.
What I had in mind that user can shutdown printing just by putting `progress_func=None` if they develop CLI applications and don't want to spam